addons/default/visiosoft/profile-module/src/Seed/UsersFieldsSeeder.php

```diff
@@ -28,166 +26,177 @@ public function run(
         AssignmentModelTranslation $assignmentModelTranslation
     )
     {
-        $namespace = 'users';
-        $locked = 0;
-        $stream = $streamRepository->findBySlugAndNamespace('users', 'users');
+        if ($this->command->confirm('This command will remove the old custom fields from the users table! Are you sure you want to continue?')) {
```

Comment:
you can use trans

Reply:
This is a command line prompt, we dont need to translate the command line do we?

Reply:
you dont need to @diashalabi  but command prompts wont work with automations like jenkins? @ozcan-durak

Reply:
Yeah I thought so, don't we have an option to reply with yes if its with jenkins? if we don't then let me remove it

Reply:
we don't as I know as least for now @diashalabi

Reply:
removed it